features.sql

```diff
@@ -297,5 +297,7 @@ CREATE TABLE features_import (
     firstremotestreamaudio character varying(255),
     firstremotestreamvideo character varying(255),
     usingsimulcast boolean,
-    sdpSemantics: character varying(32)
+    sdpSemantics: character varying(32),
+    calledAddtrack: boolean,
```

Comment:
it looks like the previous names are all lowercase "firstremotestreamaudio".  Maybe you want to maintain that style.

Reply:
redshift doesn't care afaik. Not sure about postgres... @fritzy?

Reply:
it does, thanks @argggh